### PR TITLE
add api link on variant page

### PR DESCRIPTION
**Pull request by leexgh**
image

image

add json logo next to hgvsg and link to api output

### CODE REVIEW

**Comment by inodb** [comment]
Thanks @leexgh! Wondering if it is better to put the JSON in a separate link/button right before the "all transcripts button". Because the data is not just for the hgvsg data, but it has almost all the data that we see on the page (aside from the lollipop tracks & oncokb i guess). What do you think @jjgao ?

File: src/component/variantPage/BasicInfo.tsx [hunk 1]
```diff
@@ -319,6 +321,30 @@ export default class BasicInfo extends React.Component<IBasicInfoProps> {
...
+                placement="top"
+                overlay={
+                    <span>Click to read more about the raw API response</span>
+                }
+            >
+                <Link
+                    to={
+                        '/annotation/' +
+                        this.props.variant +
+                        '?fields=hotspots%2Cmutation_assessor%2Cmy_variant_info%2Cptms%2Cannotation_summary'
```
Comment (leexgh) [inline src/component/variantPage/BasicInfo.tsx:336]: I include hotspots, mutation assessor, my variant info, ptm, annotation summary. Any other fields I need to add?

Comment (inodb) [reply src/component/variantPage/BasicInfo.tsx:336]: that looks good to me 👍

File: src/component/variantPage/BasicInfo.tsx [hunk 2]
```diff
@@ -319,6 +321,30 @@ export default class BasicInfo extends React.Component<IBasicInfoProps> {
...
+                    }
+                    target="_blank"
+                    style={{ paddingLeft: '8px', paddingRight: '8px' }}
+                >
+                    {'{JSON} '}
```
Comment (inodb) [inline src/component/variantPage/BasicInfo.tsx:341]: maybe just
```suggestion
                    {'JSON '}
```

File: src/component/variantPage/BasicInfo.tsx [hunk 3]
```diff
@@ -319,6 +321,30 @@ export default class BasicInfo extends React.Component<IBasicInfoProps> {
         );
     }
 
+    private jsonButton() {
+        return (
+            <DefaultTooltip
+                placement="top"
+                overlay={
+                    <span>Click to read more about the raw API response</span>
```
Comment (inodb) [inline src/component/variantPage/BasicInfo.tsx:329]: Maybe something like:
```suggestion
                    <span>
                            Click to view to raw API response<br /><br />
                            Click <a href="https://docs.genomenexus.org/api">here</a> for more info about the API 
                   </span>
```

**Pull request merged**